agents/builtin/mongodb/internal/profiler/aggregator/aggregator.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/percona/pmm-agent/agents/builtin/mongodb/internal/status"
 )
 
-const (
+var (
 	DefaultInterval  = time.Duration(60 * time.Second)
 	ReportChanBuffer = 1000
```

Comment:
Let's make it unexpected const.

Reply:
Don't understand. :/ What does it means `unexpected const`?

Reply:
Broken autocomplete. un-exported

Reply:
Whatever, I don't understand what exactly the problem is? Is it about some IDE or editor problem? I don't see any problems here. `go test` works fine. It compiles and runs. What should I do here?

Reply:
It is about the exported package API. We should export only what is used.

```go
const reportChanBuffer = 1000
```